dragon_gigapet/Animal.rb

```diff
@@ -0,0 +1,17 @@
+class Animal
+    attr_reader :hunger, :bored, :tired, :need_affection 
+    def initialize
+       
+        @hunger = random * 10 
+        @bored = random * 10 
+        @tired = random * 10
+        @need_affection = random * 10 
+        super
```

Comment:
I don't think `super` does anything here.

Reply:
Agreed.

Reply:
I deleted the super and cleaned up the initialize method.